src/deluge/processing/engines/audio_engine.cpp

```diff
@@ -247,7 +247,7 @@ Voice* cullVoice(bool saveVoice, bool justDoFastRelease) {
 
 	uint32_t bestRating = 0;
 	Voice* bestVoice = NULL;
-
+	Debug::println("cull");
```

Comment:
Doesn't this lead to duplicated lines? There's printing in the routine function for culls as well

Reply:
It is a duplicate that won't go in the final.